src/logging.h

```diff
@@ -77,6 +77,9 @@ namespace BCLog {
 
         std::string LogTimestampStr(const std::string& str);
 
+        /** Slots that connect to the print signal */
+        std::list<std::function<void(const std::string&)>> m_print_callbacks /* GUARDED_BY(m_cs) */ {};
```

Comment:
Why comment the lock annotation?

Reply:
It can't compile otherwise. If you include `sync.h`, the tests won't run after compilation.